packages/shared/lib/core/wallet/actions/index.ts

```diff
@@ -30,7 +30,6 @@ export * from './getNodeInfo'
 export * from './setStrongholdPasswordClearInterval'
 export * from './startBackgroundSync'
 export * from './getParticipationOverview'
-export * from './getAddressesWithOutputs'
```

Comment:
We don't, I already adapted the necessary code for the vesting outputs to continue to work

Reply:
See `packages/shared/lib/contexts/vesting/stores/selected-wallet-vesting-outputs.store.ts`, if we need to consider  more addresses, it can be easily done